pyproject.toml

```diff
@@ -147,6 +147,7 @@ sanic-routing = "^0.7.2"
 pluggy = "^1.0.0"
 slack-sdk = "^3.19.2"
 confluent-kafka = "^1.9.2"
+pytest-rerunfailures = "11.1.2"
```

Comment:
Do we need this if we're just using pytest.marky?

Reply:
No idea actually, testing it out.